django_nextjs/render.py

```diff
@@ -49,7 +49,7 @@ def _get_nextjs_request_cookies(request: HttpRequest):
     (i.e. dont use HTTP unsafe methods or GraphQL mutations).
     https://docs.djangoproject.com/en/3.2/ref/csrf/#is-posting-an-arbitrary-csrf-token-pair-cookie-and-post-data-a-vulnerability
     """
-    unreserved_cookies = {k: v for k, v in request.COOKIES.items() if not morsel.isReservedKey(k)}
+    unreserved_cookies = {k: v for k, v in request.COOKIES.items() if not morsel.isReservedKey(k) and k and v}
```

Comment:
also we can move the new condition before not `morsel.isReservedKey(k)` to be sure sure.
something like this:
```python
unreserved_cookies = {k: v for k, v in request.COOKIES.items() if k and not morsel.isReservedKey(k)}
```

Reply:
Good call, changed it!